README.md

```diff
@@ -52,6 +52,7 @@ rules.
 ## Projects using rules_xcodeproj
 
 - [Envoy Mobile](https://github.com/envoyproxy/envoy-mobile)
+- Slack
```

Comment:
Do you want to link to your BazelCon talk where you publicly say this?

Reply:
Is there prior art for the format here?

Reply:
The links in this list are either open source examples of using rules_xcodeproj, or (starting with yours, but others will be adjusted or added later) blog posts, tweets/toots, or talks announcing use.

Reply:
I'll leave it as-is for now. I want to highlight just Slack here, and not anything personal.